pyaedt/modeler/Actors.py

```diff
@@ -0,0 +1,345 @@
+from pyaedt import aedt_exception_handler
```

Comment:
Comment for file:
If we're adding new files, we should rename to lowercase.  I can understand modules being capitalized due to legacy files, but if we're making new ones we might as well start them off right and name them lowercase.  If you'd prefer, we can rename all in a separate PR to avoid upper/lower case inconsistency.